pkg/apis/intevents/v1alpha1/brokercell_types.go

```diff
@@ -68,8 +68,29 @@ var (
 	_ duckv1.KRShaped = (*BrokerCell)(nil)
 )
 
+// ComponentParameters specifies scaling and resource parameters to be used
```

Comment:
The struct only has scaling parameters currently. Is the plan to add resource parameters as well?

Reply:
My plan is to evaluate if with the scaling parameters is enough to get reliable performance benchmark results. If it is then there should be no need, afaik, to add more parameters here.